.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,24 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - master
+      - main
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_ID: ${{ secrets.CRM_DOCKER_ID }}
```

Comment:
@vasileiosC @darren-west would you be able to create these org level secrets for us?
`CRM_DOCKER_ID` and `CRM_DOCKER_PASSWORD` 🙇